x/ecocredit/dbms/schema.sql

```diff
@@ -0,0 +1,25 @@
+
+CREATE TABLE accounts (
+	address     varchar(255) PRIMARY KEY
+);
+
+CREATE TABLE credit_classes (
+	class_id    varchar(255)     PRIMARY KEY,
+	designer    varchar(255)     REFERENCES accounts ,
+	issuers     varchar(255)[]   NOT NULL,
```

Comment:
why not having another table to represent this many-to-many relationship between `accounts` and `credit_classes`?